docs/notebooks/t-test.ipynb

```diff
@@ -39,7 +39,7 @@
    "cell_type": "markdown",
```

Comment:
<p>We are using the same introductory sentence for both sections "Model 1" and "Model 2". In this first section, I would say something like "When we carry out a two sample t-test we are implicitly using a linear model that can be specified in different ways. One approach is the following"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>